src/Tags/Concerns/GetsFormSession.php

```diff
@@ -12,12 +12,7 @@ trait GetsFormSession
      */
     protected function getFormSession($formName = 'default')
     {
-        $data = [];
-
-        $errors = optional(session()->get('errors'))->getBag($formName);
-
-        $data['errors'] = $errors ? $errors->all() : [];
-        $data['error'] = $errors ? $this->getFirstErrorForEachField($errors) : [];
+        $data = $this->getErrorsFromFormSession($formName);
```

Comment:
I extracted the `getErrorsFromFormSession()` from `getFormSession` because the `{{ user:forgot_password_form }}` doesn't need all of the session data the method would provide. Due to that only the relevant data are fetched in `resetPasswordForm()`.